### DISCUSSION TITLE
Cant install Waves bundles

### DISCUSSION THREAD

**Discussion by Oemfoefoe-1984** [apohl79/audiogridder#640] [Help]
hi guys

im very excited about Soundgridder...it works fantastic!

Im building a big Win 10 server, and i have issues installing Waves plugins.
It worked fine yesterday, but because i own too much Waves plugins, i tried to delete some plugins from the plugin list because the list was just way too big ;)
Suddenly the Waves plugins dont show up anymore.

I tried re-installing a bunch of times but they dont show up. VST2 nor VST3

I tried different folders, different locations.

I see a LOT of failed plugins in red in the plugin list.
How can i delete this 'blacklist' on Win 10?

Is there a way to delete everything so i can start over? Are there any hidden files i need to delete for a clean uninstall? Any hidden config files?

Im using 1.3B, rest of my plugins work fine.

**Comment by CaBuffalo** [comment]
Try to delete the config file of AG server and rescane it again.

https://audiogridder.com/server/#config

On Wed, Jul 14, 2021 at 1:32 PM Oemfoefoe-1984 ***@***.***>
wrote:

> hi guys
>
> im very excited about Soundgridder...it works fantastic!
>
> Im building a big Win 10 server, and i have issues installing Waves
> plugins.
> It worked fine yesterday, but because i own too much Waves plugins, i
> tried to delete some plugins from the plugin list because the list was just
> way too big ;)
> Suddenly the Waves plugins dont show up anymore.
>
> I tried re-installing a bunch of times but they dont show up. VST2 nor VST3
>
> I tried different folders, different locations.
>
> I see a LOT of failed plugins in red in the plugin list.
> How can i delete this 'blacklist' on Win 10?
>
> Is there a way to delete everything so i can start over? Are there any
> hidden files i need to delete for a clean uninstall? Any hidden config
> files?
>
> Im using 1.3B, rest of my plugins work fine.
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apohl79/audiogridder/discussions/640>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ARMAR6CHN6HJTN6NWZAEL73TXXX7VANCNFSM5AMFXLJA>
> .
>


-- 
Kevin Ma
Supported by Gmail.

**Reply by Oemfoefoe-1984** [reply]
This is a bit confusing:

Waves uses 'waveshells' to redirect plugins to their actual .bundle locations.
If you deactivate plugins in the plugin list in AG, you also deactivate other plugins connected to the same waveshell. For instance, i own the CLA bundles, and it has a lot of mono plugins i will never use.
Deactivating this in AG also deactivates other plugins from Waves.
Reactivating in AG doesnt work.
You have to manually edit the .cfg file and delete the entries there.
Then the lost plugins show up again after rescan.
If you have a lot of plugins this is a bit difficult.

All i want is to organize my plugins in the AG plugin, so i dont have these big lists with bloat plugins.

Does anybody know a sollution to deactivate sertain Waves plugins?

What about some way to organize the plugin list like in Cubase, would this be a lot of work to implement?

Thanks
